scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -719,7 +719,9 @@ export function MonitoringGroup(
                             searchCriteria.source.size = PAGE_SIZE;
                         }
                     }
-                } else if (scope.group != null && scope.group.type === 'personal') {
+                } else if (scope.group != null
+                    && (scope.group.type === 'personal'
+                        || scope.group.type === 'markedForMe')) {
```

Comment:
When you implement extensions API interfaces in the core, the implementation must no rely on anything in extension code - only the interface of the API. Here you are hardcoding an ID, which means if I changed the ID in the extension, it would stop working.

To avoid that, you could check IDs of all registered extensions for personal section, similarly as you do in `filterQueryByCardType`